lib/config/options/index.ts

```diff
@@ -1472,7 +1472,7 @@ const options: RenovateOptions[] = [
     description: 'Message to use for commit messages and pull request titles.',
     type: 'string',
     default:
-      '{{{commitMessagePrefix}}} {{{commitMessageAction}}} {{{commitMessageTopic}}} {{{commitMessageExtra}}} {{{commitMessageSuffix}}}',
+      '{{{commitMessageAction}}} {{{commitMessageTopic}}} {{{commitMessageExtra}}} {{{commitMessageSuffix}}}',
```

Comment:
is this intended?

Reply:
Yeah, now it is responsibility of `SemanticCommitMessage` or `CustomCommitMessage`

Reply:
I don't think we can make this change without it being breaking

Reply:
And this is not the type of breaking where you can just add a breaking flag and move on

Reply:
In future, avoid make such design decisions without discussing the issue first, because it can lead to PRs which get rejected

Reply:
Sorry guys, it was unnecessary change. Unit tests for `generate.ts` works fine even without this change